contracts/proxy/contract.go

```diff
@@ -19,9 +25,14 @@ func OnNEP17Payment(from interop.Hash160, amount int, data any) {
 
 // nolint:deadcode,unused
 func _deploy(data any, isUpdate bool) {
+	args := data.(struct {
```

Comment:
NACK. No parameters, please. You can reuse NNS. I see there is a circular dependency, but you can push hashes into NNS before deploying other contracts (and you can do it in a single transaction).

Reply:
ok